src/abbfreeathome/devices/heating_actuator.py

```diff
@@ -0,0 +1,90 @@
+"""Free@Home HeatingActuator Class."""
+
+from typing import Any
+
+from ..api import FreeAtHomeApi
+from ..bin.pairing import Pairing
+from .base import Base
+
+
+class HeatingActuator(Base):
+    """Free@Home HeatingActuator Class."""
+
+    _state_refresh_output_pairings: list[Pairing] = [
+        Pairing.AL_INFO_VALUE_HEATING,
+    ]
+
+    def __init__(
+        self,
+        device_id: str,
+        device_name: str,
+        channel_id: str,
+        channel_name: str,
+        inputs: dict[str, dict[str, Any]],
+        outputs: dict[str, dict[str, Any]],
+        parameters: dict[str, dict[str, Any]],
+        api: FreeAtHomeApi,
+        floor_name: str | None = None,
+        room_name: str | None = None,
+    ) -> None:
+        """Initialize the Free@Home HeatingActuator class."""
+        self._state: int | None = None
+
+        super().__init__(
+            device_id,
+            device_name,
+            channel_id,
+            channel_name,
+            inputs,
+            outputs,
+            parameters,
+            api,
+            floor_name,
+            room_name,
+        )
+
+    @property
+    def state(self) -> int | None:
+        """Get the state of the actuator."""
+        return self._state
+
+    async def set_state(self, value: int):
```

Comment:
Instead of "state" would maybe "position" be a better term?

Reply:
I was of two minds about that:
- each device should have a state
- this device has a position

I will change it to position